scss/_patterns_segmented-control.scss

```diff
@@ -0,0 +1,60 @@
+@mixin vf-p-segmented-control {
+  %vf-button-group-borders {
+    border-bottom: 0;
+    border-left: 0;
+    border-right: 1px solid $color-mid-light;
+    border-top: 0;
+
+    &:last-child {
+      border-right: 0;
+    }
+  }
+
+  .p-segmented-control {
+    @extend %vf-has-round-corners;
+
+    border: 1px solid $color-mid-dark;
+    display: inline-flex;
+    margin-bottom: $spv-outer--small-scaleable;
+
+    // stylelint-disable selector-max-type
+    button,
+    [class*='p-button'] {
+      @extend %vf-button-group-borders;
+
+      border-radius: 0;
+      margin: 0;
+    }
+
+    .p-segmented-control__radio {
+      @extend %vf-button-group-borders;
+
+      margin-bottom: 0;
+      padding-top: 0;
+    }
+
+    [type='radio'] {
+      @extend %vf-hidden-tick-input;
+
+      &:checked + span {
+        background-color: $color-mid-x-light;
+      }
+
+      &:focus + span {
+        outline: $bar-thickness solid $color-focus;
+        position: relative;
+        z-index: 1;
+      }
+    }
+
+    &.is-dense {
+      button,
+      [class*='p-button'],
+      [type='radio'] + span {
+        padding-bottom: calc(#{$spv-nudge - $sp-unit * 0.5} - 1px);
```

Comment:
this si repeated in _base_buttons.scss; can you move to a placeholder and extend it from here and from the base buttton?